corehq/apps/app_manager/templates/app_manager/v2/partials/module_view_settings.html

```diff
@@ -126,6 +126,8 @@ <h4 class="panel-title panel-title-nolink">{% trans "Advanced Settings" %}</h4>
             </div>
             <div class="panel-body">
               {% include 'app_manager/v2/partials/case_list_form_setting.html' %}
+
+              {% include 'app_manager/v1/partials/case_list_setting.html' with LABEL="Case List Menu Item" DESC="An item in the module's menu that lets you browse the case list without moving on to fill out a form." SLUG="case_list" case_list=module.case_list %}
```

Comment:
seems more verbose right now, but will make people more aware of making changes and make things easier when we move over to v2